src/tooltip/tooltip.js

```diff
@@ -133,13 +133,15 @@
         }
 
         this.boundMouseEnterHandler = this.handleMouseEnter_.bind(this);
-        this.boundMouseLeaveHandler = this.handleMouseLeave_.bind(this);
+        this.boundMouseLeaveHandler = this.hideTooltip_.bind(this);
+        this.boundScrollHandler = this.hideTooltip_.bind(this);
```

Comment:
You don't need to bind the same thing twice. Simply do *one* bound handler and re-use it for each event listener. Same result, less memory.